### DISCUSSION TITLE
How can we estimate the % of observations from a given surveillance system or in a certain outbreak response context?

### DISCUSSION THREAD

**Discussion by avallecam** [epiforecasts/EpiNow2#636] [Q&A]
In [Gostic et al. 2020](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008409#sec011), in the accounting for incomplete observations, authors share the factors that change the assumption of a constant fraction of observed infections.

In [White et al 2019](https://onlinelibrary.wiley.com/doi/10.1111/j.1750-2659.2009.00106.x), in the Adjustment for changes in reporting fraction section, authors mention that changes in case ascertainment informed changes in Rt and R0 estimates.

Questions:
- From a workflow perspective, can we use [{cfr} to estimate ascertainment](https://epiverse-trace.github.io/cfr/articles/estimate_ascertainment.html) and inform on the % of observations to add in the `obs` argument in epinow()?
- Do you have examples of study evaluations on designed surveillance systems that aim to estimate this % of expected observations? 
- Or, do you usually expect to input data from other data sources?

**Comment by jamesmbaazam** [comment]
> In [Gostic et al. 2020](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008409#sec011), in the accounting for incomplete observations, authors share the factors that change the assumption of a constant fraction of observed infections.
> 
> In [White et al 2019](https://onlinelibrary.wiley.com/doi/10.1111/j.1750-2659.2009.00106.x), in the Adjustment for changes in reporting fraction section, authors mention that changes in case ascertainment informed changes in Rt and R0 estimates.
> 
> Questions:
> 
> * From a workflow perspective, can we use [{cfr} to estimate ascertainment](https://epiverse-trace.github.io/cfr/articles/estimate_ascertainment.html) and inform on the % of observations to add in the `obs` argument in epinow()?

I believe in theory, you can pass the result of {cfr} to `scale` in `obs_opts()` (with uncertainty around it) since any value you supply here is based on your assumptions/prior knowledge anyway. You must also be aware of the assumptions/limitations/interpretation of the results and underlying methods used by {cfr} to estimate ascertainment. For example, {cfr} uses "Normal approximation to binomial likelihood", from a quick glance at the linked vignette. @sbfnk Thoughts?

> * Do you have examples of study evaluations on designed surveillance systems that aim to estimate this % of expected observations?

Are you looking for something like the following:

- https://bmcpublichealth.biomedcentral.com/articles/10.1186/1471-2458-14-147#Sec12
- https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/disease-areas/covid-19/report-31-syria/
- https://www.sciencedirect.com/science/article/pii/S0022519322003241#sec2

> * Or, do you usually expect to input data from other data sources?

I'm not sure I understand this.
